### ISSUE TITLE
rocm3.5 - Could not find a package configuration file provided by "rocblas"

### ISSUE THREAD

**Issue by atamazov** [ROCmSoftwarePlatform/MIOpen#263]
Vanilla rocm3.5:
```
$ sudo cmake -P install_deps.cmake
...
$ CXX=/opt/rocm/llvm/bin/clang++ cmake -DMIOPEN_USE_COMGR=On -DBUILD_DEV=On -DCMAKE_BUILD_TYPE=debug -DMIOPEN_TEST_FLAGS=--disable-verification-cache ../..
-- The C compiler identification is GNU 7.3.0
-- The CXX compiler identification is Clang 11.0.0
-- Check for working C compiler: /usr/bin/cc
-- Check for working C compiler: /usr/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Check for working CXX compiler: /opt/rocm/llvm/bin/clang++
-- Check for working CXX compiler: /opt/rocm/llvm/bin/clang++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Found PkgConfig: /usr/bin/pkg-config (found version "0.29.1")
-- Checking for module 'sqlite3'
--   Found sqlite3, version 3.17.0
-- Found BZip2: /usr/local/lib/libbz2.a (found version "1.0.6")
-- Looking for BZ2_bzCompressInit
-- Looking for BZ2_bzCompressInit - found
-- Performing Test HAS_HIP
-- Performing Test HAS_HIP - Success
-- Performing Test HIP_CLANG_SUPPORTS_PARALLEL_JOBS
-- Performing Test HIP_CLANG_SUPPORTS_PARALLEL_JOBS - Success
-- Hip compiler flags:  -x hip --hip-device-lib-path=/opt/rocm/lib -mllvm -amdgpu-early-inline-all=true -mllvm -amdgpu-function-calls=false   -D__HIP_ROCclr__=1 -isystem /opt/rocm-3.5.0/hip/../include -isystem /opt/rocm/
llvm/lib/clang/11.0.0/include/..  -D__HIP_PLATFORM_HCC__=1  -D__HIP_ROCclr__=1 -isystem /opt/rocm-3.5.0/hip/include -isystem /opt/rocm/include --hip-device-lib-path=/opt/rocm/lib --hip-link
-- hip compiler: /opt/rocm/bin/clang-ocl
CMake Error at CMakeLists.txt:205 (find_package):
  Could not find a package configuration file provided by "rocblas" with any
  of the following names:

    rocblasConfig.cmake
    rocblas-config.cmake
```

**Comment by atamazov** [comment]
Fixed after
```
$ sudo apt-get install rocblas
Reading package lists... Done
Building dependency tree
Reading state information... Done
The following NEW packages will be installed:
  rocblas
0 upgraded, 1 newly installed, 0 to remove and 817 not upgraded.
Need to get 71.9 MB of archives.
After this operation, 3219 MB of additional disk space will be used.
Get:1 http://repo.radeon.com/rocm/apt/debian xenial/main amd64 rocblas amd64 2.22.0.2367-b2cceba [71.9 MB]
Fetched 71.9 MB in 19s (3881 kB/s)
Selecting previously unselected package rocblas.
(Reading database ... 225138 files and directories currently installed.)
Preparing to unpack .../rocblas_2.22.0.2367-b2cceba_amd64.deb ...
Unpacking rocblas (2.22.0.2367-b2cceba) ...
Setting up rocblas (2.22.0.2367-b2cceba) ...
```
Rocbals 2.22.0.2367-b2cceba released for ROCm 3.5.0, https://github.com/ROCmSoftwarePlatform/rocBLAS/releases/tag/rocm-3.5.0

**Comment by atamazov** [comment]
Nobody else complains.
